tests/unit/components/QueryBuilder.spec.ts

```diff
@@ -58,4 +60,36 @@ describe( 'QueryBuilder.vue', () => {
 		const wrapper = shallowMount( QueryBuilder, { store, localVue } );
 		expect( wrapper.find( '.querybuilder__condition-placeholder' ).exists() ).toBeFalsy();
 	} );
+
+	it( 'shows the "or" in toggle when there is two conditions', () => {
```

Comment:
a wording thingy: 
shows the "or" in toggle when there ARE two conditions

Reply:
oh right. Done.